h2o-hadoop-3/h2o-mapreduce-3-generic/build.gradle

```diff
@@ -1,6 +1,7 @@
 ext {
   hadoopVersion = 'cdh6.0.1'
   hadoopMavenArtifactVersion = '3.0.0-cdh6.0.1'
+  hiveMavenArtifactVersion = '2.1.1-cdh6.0.1'
```

Comment:
this is compileOnly it does not get bundled with the actual driver and is needed only to have HiveConf and HiveConnection classes at compile time - so even though there is different version of the class at runtime, it should be fine since the only method used is HiveConnection#getDelegationToken

Reply:
Oh, didn't see that it's compileOnly. Thanks.